### ISSUE TITLE
Sniper can shoot empty GLA Stinger Site, but not empty USA Firebase

### ISSUE THREAD

**Issue by xezon** [TheSuperHackers/GeneralsGamePatch#1590]
Sniper can shoot empty GLA Stinger Site, but not empty USA Firebase.

**Comment by xezon** [comment]
It is not clear to me what behavior is ultimately preferred.

Stinger Site behavior pro:
- Sniper and other units in group selection can click and attack Stinger Site

Stinger Site behavior contra:
- Snipers shoot on structure all the time

I was not able to streamline behavior either way.

```
Object GLAStingerSite
  Body = HiveStructureBody ModuleTag_04
    SwallowDamageTypesIfSlavesNotExisting = NONE +SNIPER +POISON
  End
End
```

Removing this entry from Stinger Site, or adding it to Firebase, changes nothing.

**Comment by MTKing4** [comment]
They're two different buildings with different properties, stinger respawns units automatically, that's why it always shoots at it, firebase is different and is loaded manually, there's no reason to standardize the two

**Comment by Stubbjax** [comment]
I think we can use some common sense here. A Pathfinder should only ever fire upon a living infantry unit.

**Comment by AlseeAns** [comment]
i don't think this is a bug, because sniper indeed snipe the empty stinger site, but make the rpg don't spawn.. right ? this is how it's work in ZH i think ?
why sniper would shot a metal thing like a firebase?
